variables.tf

```
@@ -13,6 +13,12 @@ variable "name" {
   description = "A name for the default platform admin role"
 }
 
+variable "postfix" {
+  type        = bool
+  default     = true
+  description = "Postfix the role and policy names with Role and Policy"
```

Could we make this a little more specific:

```suggestion
  description = "Append \"Policy\" and \"Role\" to the created admin role and policy names respectively"
```